app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,49 @@
+class CommentsController < ApplicationController
+
+  def index
+    @comments = Comment.all
+  end
+
+  # new
+  def new
+    @post = Post.find(params[:artist_id])
```

Comment:
`:artist_id`! Be careful about replacing alll appearances of a variable when you reuse code from other apps.